docs/Python-for-Formula-Authors.md

```diff
@@ -143,29 +143,29 @@ If the configure script takes a `--with-python` flag, it usually will not need e
 
 If the `configure` and `make` scripts do not want to install into the Cellar, sometimes you can:
 
-1. Call `./configure --without-python` (or a similar named option)
-1. `cd` into the directory containing the Python bindings
-1. Call `setup.py` with `system` and `Language::Python.setup_install_args` (as described above)
+1. call `./configure --without-python` (or a similar named option)
+2. `cd` into the directory containing the Python bindings
+3. call `setup.py` with `system` and `Language::Python.setup_install_args` (as described above)
 
-Sometimes we have to `inreplace` a `Makefile` to use our prefix for the Python bindings. (`inreplace` is one of Homebrew's helper methods, which greps and edits text files on-the-fly.)
+Sometimes we have to edit a `Makefile` on-the-fly to use our prefix for the Python bindings using Homebrew's [`inreplace`](Formula-Cookbook.md#inreplace) helper method.
 
 ## Libraries
 
 ### Python declarations
 
 Libraries built for Python 3 should include `depends_on "python"`, which will bottle against Homebrew's Python 3.x. Python 2.x libraries must function when they are installed against either the system Python or brewed Python.
 
-Python 2 libraries do not need a `depends_on "python@2"` declaration; they will be built with the system Python, but should still be usable with any other Python 2.7. If this is not the case, it is an upstream bug; [here is some advice for resolving it](https://blog.tim-smith.us/2015/09/python-extension-modules-os-x/).
+Python 2 libraries do not need a `depends_on "python@2"` declaration; they will be built with the system Python, but should still be usable with any other Python 2.7. If this is not the case, it's an upstream bug; [here's some advice for resolving it](https://blog.tim-smith.us/2015/09/python-extension-modules-os-x/).
```

Comment:
I don't really like contractions in tech writing but if there's nothing else to change in this file, leave it.

Reply:
This matches the wording in the Bindings section earlier in the document.